ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -0,0 +1,576 @@
+        echo "-----------------------------------------------------------"
+        openstack flavor list -f value; echo $?
+        echo "-----------------------------------------------------------"
+        openstack volume list -f value; echo $?
+        echo "-----------------------------------------------------------"
+
+        export EXPIRATION_DATE=$(date -d '4 hours' --iso=minutes --utc)
+        export SSH_PUB_KEY=$(cat "${SSH_PUB_KEY_PATH}")
+        export PULL_SECRET=$(cat "${PULL_SECRET_PATH}")
+
+        # move private key to ~/.ssh/ so that installer can use it to gather logs on bootstrap failure
+        mkdir -p ~/.ssh
+        cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/id_rsa
+        chmod 0600 ~/.ssh/id_rsa
+
+        # create a new floating ip tagged with CLUSTER_NAME so it can be deleted later
+        LB_FIP=$(openstack floating ip create --description $CLUSTER_NAME $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address')
+
+        echo "Creating DNS record for api.$CLUSTER_NAME.$BASE_DOMAIN. -> $LB_FIP"
+        cat > /tmp/artifacts/installer/api-record.json <<EOF
+        {
+        "Comment": "Create the public OpenShift API record",
+        "Changes": [{
+            "Action": "UPSERT",
+            "ResourceRecordSet": {
+              "Name": "api.$CLUSTER_NAME.$BASE_DOMAIN.",
+              "Type": "A",
+              "TTL": 300,
+              "ResourceRecords": [{"Value": "$LB_FIP"}]
+              }
+        }]}
+        EOF
+
+        cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1beta4
+        baseDomain: ${BASE_DOMAIN}
+        controlPlane:
+          hyperthreading: Enabled
+          name: master
+          replicas: 3
+        metadata:
+          name: ${CLUSTER_NAME}
+        platform:
+          openstack:
+            cloud:            ${OS_CLOUD}
+            externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
+            computeFlavor:    ${OPENSTACK_FLAVOR}
+            region:           ${OPENSTACK_REGION}
+            lbFloatingIP:     ${LB_FIP}
+        pullSecret: '${PULL_SECRET}'
+        sshKey: |
+          ${SSH_PUB_KEY}
+        EOF
+
+        # Create a ramdisk for the etcd storage. This helps with disk latency
+        # unpredictability in the OpenStack cloud used by the CI:
+        TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create ignition-configs --log-level=debug
+        python -c \
+            'import json, sys; j = json.load(sys.stdin); j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
+            </tmp/artifacts/installer/master.ign \
+            >/tmp/artifacts/installer/master.ign.out
+        mv /tmp/artifacts/installer/master.ign.out /tmp/artifacts/installer/master.ign
```

Comment:
We had to resort to this trick for the openstack jobs because the disks on the cloud we're using were not performant enough for etcd and caused our jobs to fails randomly in very cryptic ways.
Etcd is extremely sensitive to disk latency.

You can use fio to check if your storage is fast enough:
https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd